### PR TITLE
Add footer section

### PR DESCRIPTION
**Pull request by Martum**
### Summary
- Add footer section to landing page

### Trello card
https://trello.com/c/OQdpL4K3/3-maquetar-el-footer-de-la-pagina

### CODE REVIEW

File: app/assets/stylesheets/main.scss [hunk 1]
```diff
@@ -59,3 +59,48 @@ body {
...
+  color: #ffffff;
+  padding: 10px;
+  background: #f5c85f;
+  border: solid #ffffff 1px;
+  text-decoration: none;
+  font-size: 16px;
+  font-weight: bold;
+}
+
+.btn:hover {
```
Comment (mcallegari10) [inline app/assets/stylesheets/main.scss]: To write a hover or another action that involves a CSS class, group up this class with the "main" class, in this case .btn, and add a reference to it with an ampersand (&). Like this:

`.btn {`
`...`
`&:hover {`
`...`
`}`
`}`

File: app/assets/stylesheets/main.scss [hunk 2]
```diff
@@ -59,3 +59,48 @@ body {
...
+  border-radius: 12px;
+  border: none;
+}
+
+.rounded-input:focus {
+  outline: none;
+}
+
+
+.btn {
```
Comment (mcallegari10) [inline app/assets/stylesheets/main.scss]: Use a better name for it, or at least a full name like `.button` :stuck_out_tongue_closed_eyes:

File: app/assets/stylesheets/main.scss [hunk 3]
```diff
@@ -59,3 +59,48 @@ body {
...
+  border-radius: 7px;
+  color: #ffffff;
```
Comment (mcallegari10) [inline app/assets/stylesheets/main.scss]: Same as before, add colors to a variable :smile:

File: app/assets/stylesheets/main.scss [hunk 4]
```diff
@@ -59,3 +59,48 @@ body {
...
+  padding: 10px;
+  background: #f5c85f;
+  border: solid #ffffff 1px;
+  text-decoration: none;
```
Comment (mcallegari10) [inline app/assets/stylesheets/main.scss]: I think this is not necessary

File: app/views/shared/_footer.html.slim [hunk 5]
```diff
@@ -0,0 +1,8 @@
+.footer
+  h3 Get info on new updates
```
Comment (mcallegari10) [inline app/views/shared/_footer.html.slim]: Remember to use pipes when you write plain text :smile:

**Comment by Martum** [comment]
image

File: app/assets/stylesheets/main.scss [hunk 6]
```diff
@@ -62,3 +62,47 @@ body {
...
+    color: $white;
+    font-weight: normal;
+  }
+}
+
+.rounded-input {
+  border: none;
+  border-radius: 12px;
+  height: 35px;
+  width: 300px;
```
Comment (SKOLZ) [inline app/assets/stylesheets/main.scss]: setting a fixed width in pixels might break the layout in screens smaller than 300px (I know this is a really small screen but the same happens when using pixel widths for wider elements). To fix this you can say:
```
width: 100%;
max-width: 300px
```
that will set the element's width to 300px unless the screen is smaller. If that's the case the element will have the full screen width.

File: app/views/shared/_footer.html.slim [hunk 7]
```diff
@@ -0,0 +1,9 @@
+.footer
+  h3.footer-title
+    | Get info on new updates
+
+  form
+  .row
+    input.rounded-input type="text"
+  .row
+    input.button type="submit" value="Subscribe"
```
Comment (SKOLZ) [inline app/views/shared/_footer.html.slim]: use 
```
button.button type='submit'
  | Subscribe
```

the input way is limited to text only while the button may contain whatever you want inside of it.

**Pull request merged**